### PR TITLE
[DD-487] Update `Experiment Entry` event to satisfy new requirements

### PR DESCRIPTION
**Pull request by rsylvian**
# Description
- Send two events (one for all pages `docs-all`  and with the actual page name `docs-pageName`)
- Add support for guest users and add different metadata for logged in users vs guest users

# Reasons
After talking to Emma S. and @julianne-cci we decided to tweak the new events for better data reporting on the dashboard 

# Events

**when guest user:**
Screen Shot 2022-03-15 at 10 07 39 PM
Screen Shot 2022-03-15 at 10 07 44 PM
Screen Shot 2022-03-15 at 10 07 58 PM

**when logged in user:**
Screen Shot 2022-03-15 at 10 09 41 PM
Screen Shot 2022-03-15 at 10 09 48 PM
Screen Shot 2022-03-15 at 10 10 06 PM

### CODE REVIEW

**Comment by rsylvian** [comment]
Adding @jtmcg & @royvandewater since @julianne-cci wanted to make sure these events work with what you had in mind for the dashboard.

File: src/js/services/trackExperimentEntry.js [hunk 1]
```diff
@@ -4,16 +4,47 @@ import { v4 as uuidv4 } from 'uuid';
...
     if (userId && orgId) {
-      const properties = {
-        id: uuidv4(),
-        timestamp: new Date().toISOString(),
+      properties = {
+        ...properties,
         userId,
-        orgId,
-        pageName: 'docs',
+        orgId: orgId,
```
Comment (tshen815) [inline src/js/services/trackExperimentEntry.js:22]: could we just do `orgId,` here?

**Review by tshen815 — APPROVED** [review]

File: src/js/services/trackExperimentEntry.js (hunk 1 above)
Comment (rsylvian) [inline src/js/services/trackExperimentEntry.js:22]:
```suggestion
        orgId,
```

Comment (rsylvian) [reply src/js/services/trackExperimentEntry.js]: great catch!

File: src/js/services/trackExperimentEntry.js [hunk 2]
```diff
@@ -4,16 +4,47 @@ import { v4 as uuidv4 } from 'uuid';
 
 const trackExperimentEntry = () => {
   let orgId = Cookies.get(COOKIE_KEY) ?? null;
-  window.OptimizelyClient.getUserId().then((userId) => {
+  const anonymousId = window.OptimizelyClient.getAnonymousId();
+
+  window.OptimizelyClient.getUserId().then((userData) => {
+    const userId = userData.id;
```
Comment (rsylvian) [inline src/js/services/trackExperimentEntry.js:10]:
```suggestion
  window.OptimizelyClient.getUserId().then((userId) => {
```

**Pull request merged**